src/main/java/graphql/Enums.java

```diff
@@ -0,0 +1,35 @@
+package graphql;
+
+import graphql.language.Description;
+import graphql.language.EnumTypeDefinition;
+import graphql.language.EnumValueDefinition;
+
+/**
+ * The enums that are understood by graphql-java
+ */
+public class Enums {
+    public static final String ON_ERROR = "OnError";
+    public static final String ON_ERROR_PROPAGATE = "PROPAGATE";
+    public static final String ON_ERROR_NULL = "NULL";
```

Comment:
I don't think the string values need to be public per se

Reply:
I am worried about the `NULL`  name here - its an enum value hence it can be

```
@errorHandling(onError : NULL)` or `@errorHandling(onError : PROPOGATE)
```

`null` is a keyword in graphql language - I think this should be 

* `ALLOW_NULL`

or perhaps the two should be

* `PROPOGATE_NULL`
* `PROPOGATE_ERROR`

which says more about whats to happen

Reply:
yeah

```
def sdl = """
            type Query {
                f(arg1 : AnEnum arg2 : AnEnum) : String
            }
            
            enum AnEnum {
                NULL
                PROPOGATE
            }    
        """

        DataFetcher df = {DataFetchingEnvironment env ->
            println "arg1=" + env.getArgument("arg1")
            println "arg2=" + env.getArgument("arg2")
            return "ok"
        }

        def graphQL = TestUtil.graphQL(sdl, [Query: [f: df]]).build()

        when:
        def er = graphQL.execute("""query q { f(arg1 : NULL arg2 : null) }""")

        then:
        er.errors.isEmpty()
```

reports

```
arg1=NULL. (A string "NULL")
arg2=null
```

I think this is very confusing